src/expressions/closure-expr.md

```diff
@@ -10,7 +10,7 @@
 > &nbsp;&nbsp; _ClosureParam_ (`,` _ClosureParam_)<sup>\*</sup> `,`<sup>?</sup>
 >
 > _ClosureParam_ :\
-> &nbsp;&nbsp; [_Pattern_]&nbsp;( `:` [_Type_] )<sup>?</sup>
+> &nbsp;&nbsp; [_OuterAttribute_]<sup>\*</sup>\ [_Pattern_]&nbsp;( `:` [_Type_] )<sup>?</sup>
```

Comment:
This seems to be rendering the `\` in "View file" for some reason... @ehuss maybe knows...?